simde/arm/neon/get_lane.h

```diff
@@ -276,7 +276,7 @@ simde_vgetq_lane_f16(simde_float16x8_t v, const int lane)
   simde_float16_t r;
 
   #if defined(SIMDE_ARM_NEON_A32V7_NATIVE) && defined(SIMDE_ARM_NEON_FP16)
-    SIMDE_CONSTIFY_8_(vget_lane_f16, r, (HEDLEY_UNREACHABLE(), SIMDE_FLOAT16_VALUE(0.0)), lane, v);
+    SIMDE_CONSTIFY_8_(vgetq_lane_f16, r, (HEDLEY_UNREACHABLE(), SIMDE_FLOAT16_VALUE(0.0)), lane, v);
```

Comment:
Odd, the tests already covered `lane` values above `3`; why didn't that cause a problem already?

@yyctw Can you take a look at https://github.com/simd-everywhere/simde/blob/d08d67cf1d9b80fcde34a0ca846f572b590edd45/test/arm/neon/get_lane.c#L505-L518 and investigate why this wasn't caught earlier? Feels like this code path is not being run by the tests..

Reply:
Based on my previous conclusion [[ref](https://github.com/simd-everywhere/simde/pull/1075#discussion_r1358220373)], even if the `HEDLEY_UNREACHABLE()` is executed during the testing process (when `lane > 3`), it will not cause the test to fail.